pac4j-core/src/main/java/org/pac4j/core/exception/RequiresHttpAction.java

```diff
@@ -92,6 +92,19 @@ public static RequiresHttpAction unauthorized(final String message, final WebCon
     }
     
     /**
+     * Build a configurable response..
+     * 
+     * @param message message
+     * @param errorCode status code of response
+     * @param context context
+     * @return a configurable response
+     */
+    public static RequiresHttpAction configurable(final String message, int errorCode, final WebContext context) {
```

Comment:
This already exists in 1.9: https://github.com/pac4j/pac4j/blob/master/pac4j-core/src/main/java/org/pac4j/core/exception/RequiresHttpAction.java#L32